### ISSUE TITLE
Revisions tab: lang attribute missing value

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#89]
URL: https://hcdev.openplus.ca/fr/node/1109/revisions
User Agent: Chrome, Firefox
Language: Both

Issue Description:  On Revisions tab, wherever user name appears, there is lang attribute which has empty strings. Empty strings suggest that the language is unknown. 

Steps to reproduce: Viewed source code; WAVE toolbar

Expected results: Language at the username should be language of the page. This attribute here is not necessary and should be removed. 

![image](https://user-images.githubusercontent.com/14280062/129408556-7b0c272c-2bd0-4f77-9dc8-9a1432dacee0.png)

![image](https://user-images.githubusercontent.com/14280062/129408624-a8d65d96-35c2-4e73-833e-03d242d90db3.png)

**Comment by qanitah** [comment]
This is still not fixed. The more revisions a page has, the more rows of versions there are. Each version has info on who made the change for that version and it has lang="" in the code. This code means that the language is either missing or invalid. This attribute (lang="") should be just removed and the language or the page will be applied to this content.

**Comment by echouin** [comment]
From OP: 
All these things with revisions are from the diff module. We don't want to get into overriding upstream code unless absolutely necessary. I would like to have this issue closed for that reason.
﻿
﻿There are other issues with this module the accessibility team has found. If it is a priority, and if there is sprint time available, we can look at fixing the module and contributing the work back so users in the future don't have these problems. This work, if desired, wound be part of the post Nov. 11 sprints﻿.

@qanitah marked as 'resolved' for MVP requirements.
